notebooks/working/Causal_Graph_Tim_Data.py

```diff
@@ -0,0 +1,446 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.3.4
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# +
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from causalgraphicalmodels import CausalGraphicalModel, StructuralCausalModel
+import pylogit
+from collections import OrderedDict
+import pylogit as cm
+from functools import reduce
+
+
+
+import os
+os.listdir('.')
+# -
+
+data = pd.read_csv('spring_2016_all_bay_area_long_format_plus_cross_bay_col.csv')
```

Comment:
Adjust the data path to read from `data` directory (so the notebook can actually be executed without error).